versions/1/manifest.spec.yml

```
@@ -273,6 +273,25 @@ spec:
           pattern: '^(([a-zA-Z0-9-]+)|([a-zA-Z0-9-]+\/[a-zA-Z0-9-]+))$'
       required:
       - github
+    buildDependencies:
```

Use snake_case for coherence with other fields.
```suggestion
    build_dependencies:
```

---

I will follow the other approach (with `build` object).